topology/graph/indexer.go

```diff
@@ -203,14 +203,9 @@ func NewMetadataIndexer(g *Graph, listenerHandler ListenerHandler, m ElementMatc
 		indexes: indexes,
 		Indexer: NewIndexer(g, listenerHandler, func(n *Node) (kv map[string]interface{}) {
 			if match := n.MatchMetadata(m); match {
-				switch len(indexes) {
-				case 0:
-					return map[string]interface{}{string(n.ID): nil}
-				default:
-					kv = make(map[string]interface{})
-					if values, err := getFieldsAsArray(n, indexes); err == nil {
-						kv[indexer.Hash(values...)] = values
-					}
+				kv = make(map[string]interface{})
```

Comment:
Why this change ?

Reply:
because when running `indexer.Get()` I want all the nodes meeting the criteria, which was not the case with the previous implementation.

Reply:
If I'm correct, this is only used for the cluster indexer

Reply:
correct

Reply:
I'm fine with it, it wasn't used anywau